xref-token/README.md

```diff
@@ -38,14 +40,20 @@ pub struct ContractMetadata {
     pub owner_id: AccountId,
     /// backend locked token id
     pub locked_token: AccountId,
-    /// reward token that haven't distribute yet
+    /// at prev_distribution_time, reward token that haven't distribute yet
     pub undistribute_reward: U128,
-    /// backend locked token amount
+    /// at prev_distribution_time, backend staked token amount
     pub locked_token_amount: U128,
+    // at call time, the amount of undistributed reward
+    pub cur_undistribute_reward: U128,
+    // at call time, the amount of backend staked token
+    pub cur_locked_token_amount: U128,
     /// XREF token supply
     pub supply: U128,
-    /// previous reward distribution time in nano secs
-    pub prev_distribution_time: u64,
+    /// previous reward distribution time in secs
+    pub prev_distribution_time_in_sec: u32,
```

Comment:
why change to second?

Reply:
Seconds is enough for the whole logic, meanwhile saving 4 bytes won't be bad after all.

Reply:
Yeah, that's true. I prefer leave that problem into 80 years later. :)